src/raft.c

```diff
@@ -376,6 +392,7 @@ RedisModuleCallReply *RaftExecuteCommandArray(RedisRaftCtx *rr,
     RedisModuleCallReply *reply = NULL;
     RedisModuleUser *user = NULL;
     RedisModuleCtx *ctx = req ? req->ctx : rr->ctx;
+    bool is_multi_session = false;
```

Comment:
The name is a little bit confusing - it's not a multi session, but a multi session with a valid persist client

Reply:
I guess in my head this is a multi using session support, not a multi without session support.  i.e. is_multi would be if just a multi is multi_session is a multi when we are handling a session.